incubator/hnc/pkg/controllers/hierarchy_controller_test.go

```diff
@@ -19,6 +19,7 @@ var _ = Describe("Hierarchy", func() {
 	var (
 		fooName string
 		barName string
+		bazName string
```

Comment:
If you're creating a new global variable, you should initialize it in `BeforeEach` as well.

Reply:
since i use different function to create ns, i remove bazName from global and declare locally.